docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -526,15 +555,58 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
    To create a RHEL-based image, run `image-builder` with the following options:
 
       * `--os`: `redhat`
-      * `--os-version`: `8` (default: `8`)
+      * `--os-version`: `8`, `9` (default: `8`)
       * `--hypervisor`: For vSphere use `vsphere`
-      * `--release-channel`: Supported EKS Distro releases include 1-27, 1-28, 1-29, 1-30 and 1-31.
+      * `--release-channel`: Supported Kubernetes versions, from 1-28 to 1-34.
       * `--vsphere-config`: vSphere configuration file (`vsphere.json` in this example)
 
       ```bash
       image-builder build --os redhat --hypervisor vsphere --release-channel 1-31 --vsphere-config vsphere.json
       ```
 
+2. Create an Ubuntu or Redhat image with vsphere-clone builder:
+   [Vsphere clone builder](https://developer.hashicorp.com/packer/integrations/hashicorp/vsphere/latest/components/builder/vsphere-clone) support has been added to EKS-A image-builder in v0.7.0. The vsphere clone builder allows user to clone an existing virtual machine/OVF template on vCenter and convert it into an EKS-A node image.
```

Comment:
```suggestion
   [Vsphere clone builder](https://developer.hashicorp.com/packer/integrations/hashicorp/vsphere/latest/components/builder/vsphere-clone) support has been added to EKS-A image-builder in v0.7.0. The vsphere clone builder allows user to clone an existing virtual machine/OVF template on vCenter and convert it into an EKS Anywhere node image.
```
I was told by marketing a long time ago to always spell out "EKS Anywhere", and only use EKS-A, eksa, or eks-a if it is literally used as an option or part of a URL, or similar.